treeherder/config/settings.py

```diff
@@ -506,7 +506,7 @@ def server_supports_tls(url):
     "PULSE_DATA_INGESTION_SOURCES",
     default=[
         # {
-        #     "name": "exchange/taskcluster-treeherder/v1/jobs",
+        #     "exchange": "exchange/taskcluster-treeherder/v1/jobs",
         #     "projects": [
         #         'mozilla-central',
         #         'mozilla-inbound'
```

Comment:
I note these projects are `mozilla-inbound.#` in RTD for setting up for local pulse ingestion, I gather they need to be sans hash here, but noting it in case it could be adjusted also.

Reply:
I think this is where the changes should be made.  We should un-comment the defaults for both the ``PULSE_DATA_INGESTION_SOURCES`` and ``PULSE_PUSH_SOURCES``.  But change them so they default to ``#`` for both ``projects`` and ``destinations``.  This way, the user only has to set their config url in their env if they just want "everything".

Reply:
Though remove the "treeherder-test'' entry.  Just use the "taskcluster-treeherder" one.

Reply:
Done. Hopefully correctly.